### PR TITLE
RNA Seq Analysis test

### PR DESCRIPTION
**Pull request by Norore**
You will found, on this push request, my answers to your test of RNA Seq Analysis. I really don't have habit to make this kind of exercise but I really learn a lot! Thank you very much.

### CODE REVIEW

**Comment by drosofff** [comment]
Thank @Norore 👍 . I will review the pull request and I may come back here for further comments

File: analysis.02/analysis.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+To substract firsts 8,000,000 reads, we will use a shell command:
+
+`
+tail -n 8000000 ERR99055{7..9}.fastq > ERR99055{7..9}s.fastq`
+
+`tail -n 8000000 ERR990560.fastq > ERR990560s.fastq`
+
+## 4. Align dataset to genome reference
+
+To align our datasets, we will use Galaxy software from https://usegalaxy.org/
```
Comment (drosofff) [inline analysis.02/analysis.md:25]: Good. Could you share your Galaxy history in usegalaxy with me (drosofff@gmail.com) ?

**Comment by drosofff** [comment]
@Norore In addition to my previous request (sharing you Galaxy history), I informed you that contrary to what I said, I will not merge your pull request: I realized that it is much more convenient to keep the test as a non-merged PR.
Thank you again for you effort

**Comment by Norore** [comment]
@drosofff Sorry for not having seen your requests before! I didn't received any notifications. I just shared my history with you on usegalaxy.

**Comment by drosofff** [comment]
Thanks @Norore it's all good

**Pull request closed without merging**